spec/features/reviews/reviews_create_spec.rb

```diff
@@ -0,0 +1,56 @@
+require "rails_helper"
+
+feature "authenticated user(rider) can review a driver", %{
+  As an authenticated user
+  I want to review a driver
+  So that I can share my ride expirience with a particular driver
+  } do
+
+  let!(:driver) { create(:driver) }
+  let!(:rider) { create(:rider) }
+  let!(:trip) { create(:trip, driver_id: driver.id) }
+  let!(:review) { create(:review,
```

Comment:
Avoid using `{...}` for multi-line blocks.<br>Block body expression is on the same line as the block start.